assets/course-theme/blocks/lesson-blocks/module-title/module-title-edit.js

```diff
@@ -7,5 +7,5 @@ import { useBlockProps } from '@wordpress/block-editor';
 export const ModuleTitleEdit = () => {
 	const blockProps = useBlockProps();
 
-	return <h3 { ...blockProps }>{ __( 'MODULE', 'sensei-lms' ) }</h3>;
+	return <div { ...blockProps }>{ __( 'Module', 'sensei-lms' ) }</div>;
```

Comment:
Why are using div instead of h3?

Reply:
I don't recall. Switched back [here](https://github.com/Automattic/sensei/pull/6783/commits/cfb327197d734f03839b497579a80af6180d4fed).